schemas/spdx-schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema" : "http://json-schema.org/draft-07/schema#",
+  "$schema" : "https://json-schema.org/draft/2019-09/schema#",
```

Comment:
This had to get updated because the `2019-09` schema is the first to support the `deprecated` property.